x-pack/plugins/observability/public/components/app/section/apm/index.tsx

```diff
@@ -19,36 +19,39 @@ import { FETCH_STATUS, useFetcher } from '../../../../hooks/use_fetcher';
 import { ChartContainer } from '../../chart_container';
 import { StyledStat } from '../../styled_stat';
 import { onBrushEnd } from '../helper';
-
-interface Props {
-  absoluteTime: { start?: number; end?: number };
-  relativeTime: { start: string; end: string };
-  bucketSize?: string;
-}
+import { useQueryParams } from '../../../../hooks/use_query_params';
+import { calculateBucketSize } from '../../../../pages/overview';
```

Comment:
If you decided to calculate the bucketSize on each App Section, I think you could move the `calculateBucketSize ` outside the `pages/overview` to a utils function.

Reply:
Fixed here https://github.com/elastic/kibana/pull/79388/commits/a04208560b58f20c5056c36d3efd59cf9b614ffc